### ISSUE TITLE
do not repeatedly read the clock

### ISSUE THREAD

**Issue by tijn** [brightin/today#2]
Awesome work! I didn't feel like making the effort to get it working again after Google disabled their old API. "Hulde!"

Anyhow, I found this small-ish isue in the method pasted below. You're repeatedly reading the clock, which will produce funny results when you run the app at midnight exactly. You won't; so it's not a big deal, but the code could be improved so easily that I really think you should.

``` ruby
  def today(opts = {})
    events(
      {
        orderBy: 'startTime',
        timeMin: Time.new(Time.now.year, Time.now.month, Time.now.day, 0, 0, 0).iso8601,
        timeMax: Time.new(Time.now.year, Time.now.month, Time.now.day, 23, 59, 59).iso8601
      }.merge(opts)
    )
  end
```

**Comment by florish** [comment]
Thanks! Was your intent to paste improved code, or just the code as it is now? It seems to be the latter, and I'm not sure how the improved code should look... Some (pseudo) code would be greatly appreciated!

**Comment by florish** [comment]
Ah right, you probably mean we should store  `Time.now` in a local variable and use that for calculating `timeMin` and `timeMax`.

**Comment by tijn** [comment]
Yes! That's it. You really want to use the same point in time everywhere in the program for each run.

**Comment by florish** [comment]
Fix commited in 9ef158e, closing this issue.
